pandas-stubs/_typing.pyi

```diff
@@ -729,8 +729,8 @@ InterpolateOptions: TypeAlias = Literal[
 ReplaceMethod: TypeAlias = Literal["pad", "ffill", "bfill"]
 SortKind: TypeAlias = Literal["quicksort", "mergesort", "heapsort", "stable"]
 NaPosition: TypeAlias = Literal["first", "last"]
-JoinHow: TypeAlias = Literal["left", "right", "outer", "inner"]
-MergeHow: TypeAlias = JoinHow | Literal["cross"]
+JoinHow: TypeAlias = Literal["left", "right", "outer", "inner", "cross"]
+MergeHow: TypeAlias = JoinHow
```

Comment:
This fix is not correct, because `JoinHow` is used as the type of the `join` argument in the `align()` method.

The right fix is to change the typing of `DataFrame.join()` to use the existing `MergeHow` type.

You would then add a test like your example that shows that `"cross"` is accepted as a value for the `how` argument of `DataFrame.join()`

Reply:
Thank you for the quick feedback, I've done as you said.